package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "formik": "^2.2.9",
```

Comment:
i do think formik is a bit overkill for this, but it is what it is :p

Reply:
It just made things easier